app/components/ui/atoms/PersonPod.js

```diff
@@ -107,12 +117,56 @@ const buildChooserText = ({ role, isRequired, ...props }) => (
   </Flex>
 )
 
-const ChoosePersonPod = props => {
+const SelectButton = ({ onIconClick, ...props }) => {
   const ChooserText = buildChooserText(props)
-  return <PersonPod icon={PlusIcon} textContainer={ChooserText} />
+  return (
+    <PersonPodContainer
+      icon={PlusIcon}
+      onIconClick={onIconClick}
+      textContainer={ChooserText}
+    />
+  )
+}
+
+PersonPodContainer.propTypes = {
+  onIconClick: PropTypes.func.isRequired,
+  textContainer: PropTypes.element.isRequired,
+  icon: PropTypes.element.isRequired,
+}
+
+switchIcon.PropTypes = {
```

Comment:
Isn't it `propTypes` with lower case first letter? But also, `switchIcon` is not a component (it's a function that returns a component) so I don't think it makes sense to have prop types at all.